go/libraries/doltcore/mvdata/data_mover.go

```diff
@@ -36,12 +36,14 @@ type MoveOperation string
 
 const (
 	OverwriteOp MoveOperation = "overwrite"
+	ReplaceOp   MoveOperation = "replace"
 	UpdateOp    MoveOperation = "update"
 	InvalidOp   MoveOperation = "invalid"
 )
 
 type CsvOptions struct {
-	Delim string
+	Delim     string
+	TableName string
```

Comment:
I don't think we should be adding TableName to the csv options.  You should be able to read and write csvs without having a table at all.